src/software/ai/hl/stp/tactic/chip_tactic_test.cpp

```diff
@@ -2,15 +2,274 @@
 
 #include <gtest/gtest.h>
 
+#include "software/ai/hl/stp/action/chip_action.h"
 #include "software/test_util/test_util.h"
 
-TEST(ChipTacticTest, constructor_test)
+TEST(ChipTacticTest, getChipDirection)
```

Comment:
For the `getChipDirection`, `getChipOrigin`, and `getChipDistanceMeters` tests, these are really just testing the new "getter" functions of the Action and so should be part of the unit tests for the `ChipAction` instead of the `ChipTactic`

The `ChipTactic` test should focus on Tactic-level behaviour like your other tests, like making sure we can chip into each quadrant.

Reply:
Should I add these to the `ChipAction` then? Or simply remove them from `ChipTactic` and call it there?

Reply:
If they are already in the `ChipActionTest` they can be removed, otherwise they should be moved to the `ChipActionTest`

Reply:
Moved to `ChipActiontest`